src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/custom.py

```diff
@@ -13,6 +15,10 @@
 # ISO format with explicit indication of timezone
 DATE_TIME_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
 
+# Autoscale settings parameter scaffold file path
+CURR_DIR = os.path.dirname(os.path.realpath(__file__))
+AUTOSCALE_SETTINGS_PARAMETER_FILE_PATH = os.path.join(CURR_DIR,
+                                                      'autoscale-parameters-template.json')
```

Comment:
@tjprescott newbie question for python: If we move it inside method wouldn't it be initialized every time we call that command?